src/styles/style.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Montserrat&display=swap');
+
+html, body {
+    margin: 0;
+    padding: 0;
+    background-color: #01223A;
+}
+
+.project-card.col1 {
+    background-color: #2B465A;
+  }
+.project-card.col2 {
+    background-color: #122B3D;
+  }
+.project-card.col3 {
+    background-color: #01223A;
+  }
+
+.project-card {
+    display: flex;
+    flex-direction: column; 
+    align-items: center;
+    justify-content: center; 
+    height: 100vh;
+}
+
+.text-container { 
+    font-family: "Lato";
+    color: white;
+    padding: 2% 0;
+    box-sizing: border-box;
+    text-align: center;
+}
+
+.project-title {
+    margin: 20px 0; 
+    font-size: calc(2vw + 10px);
+    text-align: left;
+    font-weight: 900;
+
+}
+
+.project-image {
+    overflow: hidden visible;

```

Comment:
visible is default value, in those cases please use overflow-x or overflow-y for other one